src/odb/include/odb/dbStream.h

```diff
@@ -199,13 +200,12 @@ class dbOStream
     return *this;
   }
 
-  template <class T1, class T2>
-  dbOStream& operator<<(const std::unordered_map<T1, T2>& m)
+  template <class T1>
+  dbOStream& operator<<(const std::vector<T1>& m)
```

Comment:
it would be better to add std::vector rather than replace std::unordered_map.  I believe that was added recently for some work that is happening in dft.